src/main/java/graphql/schema/AbstractGraphQLType.java

```diff
@@ -0,0 +1,32 @@
+package graphql.schema;
+
+import static graphql.Assert.assertNotNull;
+
+public class AbstractGraphQLType implements GraphQLType {
```

Comment:
I'm not really about adding AbstractGraphQLType with the only purpose is to not having duplicate